### DISCUSSION TITLE
Basic Firearm

### DISCUSSION THREAD

**Discussion by AFK-TankyMOFO** [FakeFishGames/Barotrauma#8540] [Ideas and suggestions]
Gunsmith talent should unlock recipes for Revolver, Riot Shotgun, and SMG.

Or their recipe should just be unlocked by default.

On its own, without recipe provided by other talent, the only weapon Gunsmith affects are Stun Gun, Stun Baton, Harpoon Gun, Knife. None of them are actually gun.

**Comment by GenericGoose** [comment]
this also sucks because they have quality levels, but since you can't craft them, you'll never get a good one.
